xcode/Subconscious/Shared/Components/HomeProfileView.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  HomeProfileView.swift
+//  Subconscious
+//
+//  Created by Ben Follington on 19/9/2023.
+//
+
+import SwiftUI
+import os
+import ObservableStore
+import Combine
+
+struct HomeProfileNavigationView: View {
+    @ObservedObject var app: Store<AppModel>
+    @ObservedObject var store: Store<HomeProfileModel>
+    var detailStack: ViewStore<DetailStackModel> {
+        store.viewStore(
+            get: HomeProfileDetailStackCursor.get,
+            tag: HomeProfileDetailStackCursor.tag
+        )
+    }
+
+    var body: some View {
+        DetailStackView(
+            app: app,
+            store: detailStack
+        ) {
+            VStack(spacing: 0) {
+                UserProfileDetailView(
+                    app: app,
+                    description: UserProfileDetailDescription(address: Slashlink.ourProfile),
+                    notify: Address.forward(
+                        send: detailStack.send,
+                        tag: DetailStackAction.tag
+                    )
+                )
+            }
+        }
+    }
+}
+
+//  MARK: View
+struct HomeProfileView: View {
+    @ObservedObject var app: Store<AppModel>
+    @StateObject private var store = Store(
+        state: HomeProfileModel(),
+        environment: AppEnvironment.default
+    )
+
+    var body: some View {
+        ZStack {
+            HomeProfileNavigationView(
+                app: app,
+                store: store
+            )
+        }
+        .onAppear {
+            store.send(.appear)
+        }
+        /// Replay app actions on local store
+        .onReceive(
+            app.actions.compactMap(HomeProfileAction.from),
+            perform: store.send
+        )
+        .onReceive(store.actions) { action in
+            let message = String.loggable(action)
+            HomeProfileModel.logger.debug("[action] \(message)")
```

Comment:
As of #906 I'm adopting the convention of giving each source of truth store's actions a static `logger` property that is exclusively used to log actions.

I'm also using `String(describing:)` for all action logging to reduce the surface area we need to cover to redact sensitive actions.

Reply:
Updated to use this approach.